libnetwork/drivers/bridge/bridge.go

```diff
@@ -399,7 +398,9 @@ func (d *driver) configure(option map[string]interface{}) error {
 		// Make sure on firewall reload, first thing being re-played is chains creation
 		iptables.OnReloaded(func() {
 			logrus.Debugf("Recreating iptables chains on firewall reload")
-			setupIPChains(config, iptables.IPv4)
+			if _, _, _, _, err := setupIPChains(config, iptables.IPv4); err != nil {
```

Comment:
hehe, I recalled the lib network code when I enabled the `dogsled` linter (best name ever); https://github.com/alexkohler/dogsled